src/figuresClasses.ts

```diff
@@ -1,11 +1,82 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: Function;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape = 'triangle',
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('Invalid parameters.');
```

Comment:
[CODE STYLE] The error message could be more descriptive. Instead of 'Invalid parameters.', you could specify what makes the parameters invalid, such as 'Triangle side lengths must be greater than zero.'